web/webpack.config.js

```diff
@@ -8,13 +8,11 @@ const CssMinimizerPlugin = require('css-minimizer-webpack-plugin');
 const HtmlMinimizerPlugin = require("html-minimizer-webpack-plugin");
 const CompressionPlugin = require("compression-webpack-plugin");
 const ESLintPlugin = require('eslint-webpack-plugin');
-const CockpitPoPlugin = require("./src/lib/cockpit-po-plugin");
 const CockpitRsyncPlugin = require("./src/lib/cockpit-rsync-plugin");
```

Comment:
Perhaps we can get rid of this one too.

Reply:
yeap, on my list to check what it exactly do and why we need it.

Reply:
I also checked if we need that manifests.js and manifest.json , but I worry it enlist list of provided localization. So probably I will skip it for now...but rsync looks ready for review as it is mainly for cockpit plugin development https://github.com/cockpit-project/cockpit/commit/e224a0296cd25a448d70810a4c114bac574b82dd

Reply:
Yes, we still need the `manifest.json` file. In the future we could have a customized version, but we need at least the languages list.